src/services/EdFi.DataManagementService.Api/Core/Model/ResourceInfo.cs

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Api.ApiSchema.Model;
+
+namespace EdFi.DataManagementService.Api.Core.Model;
+
+/// <summary>
+/// Base API resource information for passing along to backends.
+/// </summary>
+public record BaseResourceInfo(
```

Comment:
Split into base and extended ResourceInfo, because it will be necessary for reference extraction shortly.